### PR TITLE
feat: node interfaces + migration

### LINKED ISSUE #34: Add storage migration for changed node object
https://github.com/threefoldtech/tfchain_pallets/commit/bf31b9fa9172513f2458fd6a05bf2661a6f31060 added an interfaces field on a node, we need a storage migration before we can put this live.

### PR DESCRIPTION
**Pull request by DylanVerstraete**

### CODE REVIEW

File: pallet-tfgrid/src/migration.rs [hunk 1]
```diff
@@ -0,0 +1,83 @@
+use super::*;
+use frame_support::weights::Weight;
+use codec::{Decode, Encode};
+
+#[derive(PartialEq, Eq, PartialOrd, Ord, Clone, Encode, Decode, Default, Debug)]
+pub struct OldNode {
```
Comment (muhamadazmy) [inline pallet-tfgrid/src/migration.rs]: can we call it NodeV1 for example so in case we have more migrations to do 🤷🏼  it will be hard to track the names.

**Comment by DylanVerstraete** [comment]
Let's try and get this live on devnet?

**Comment by DylanVerstraete** [comment]
PR inlcudes now:

Node objects:

- Added field `interfaces`
- Removed field `uptime`

**Review by robvanmieghem — APPROVED** [review]
lgtm

**Pull request merged**